docs/reference/feel/language-guide/language-guide-introduction.md

```diff
@@ -0,0 +1,16 @@
+---
+id: language-guide-introduction 
+title: Introduction
+---
+
+Everything starts with the [Data Types](./feel-data-types.md). Read this first to have a better
+understanding of the other content.
```

Comment:
Read what first for a better understanding of which content? Just trying to clarify here.

Reply:
The user should read first about data types before continuing with other chapters, like expressions or built-in functions.

Reply:
Adjusted in latest commit.